plugins/catalog-backend/migrations/20201007201501_index_entity_search.js

```diff
@@ -21,8 +21,11 @@
  */
 exports.up = async function up(knex) {
   await knex.schema.alterTable('entities_search', table => {
-    table.index(['key'], 'entities_search_key');
-    table.index(['value'], 'entities_search_value');
+    const options = knex.client.config.client.includes('mysql')
+      ? { indexType: 'FULLTEXT' }
```

Comment:
Are you sure about this? The database is meant to only ever support exact matches against the entire key and value. I'm wondering what this does to the matching logic. 🤔

Reply:
I'll revisit this. As it stands, MySQL will not work without defining the indexType as "FULLTEXT". I don't know if that's down to how Backstage's search is currently implemented, and/or if it's a difference in how Postgres interprets a match.

Reply:
Heheh. Figured it out why FULLTEXT works. In MySQL it'll let you index across blobs (which I didn't know), and normally you can't. I thought `value` and `key` were already a string when I looked at the DB. Turns out, I missed one:

https://github.com/backstage/backstage/blob/master/plugins/catalog-backend/migrations/20200807120600_entitySearch.js#L26-L38

line 26 `table.string...`
line 26 `table.string...`